MAINTAINERS.md

```diff
@@ -49,10 +57,20 @@ or this project can be selected from Visual Studio and click Run.
 > nuget pack Minio.core.nuspec
 ... package built ...
 ```
+#### Upload the package to nuget.org
+Login to nuget.org(https://www.nuget.org) and find the API Key for Minio
+```powershell
+>  export MINIO_API_KEY=??
+>  nuget setApiKey $MINIO_API_KEY
+>  nuget push .\Minio.1.0.2.nupkg -Apikey $MINIO_API_KEY -src https://nuget.org
+>  nuget push .\Minio.NetCore.1.0.2.nupkg -Apikey $MINIO_API_KEY -src https://nuget.org
+
+```
+#### Publish package to nuget.org
+Login to nuget.org to [publish](https://www.nuget.org/account/Packages)  the new package versions through browser.
```

Comment:
Is this necessary can this be automated?

Reply:
yes, replaced this with a verification step.